packages/theming/theme-tokens/src/getMacOSAliasTokens.ts

```diff
@@ -0,0 +1,26 @@
+import macOSLightAliasTokens from './light-macos/tokens-alias';
+import macOSDarkAliasTokens from './dark-macos/tokens-alias';
+import macOSLightHCAliasTokens from './light-high-contrast-macos/tokens-alias';
+import macOSDarkHCAliasTokens from './dark-high-contrast-macos/tokens-alias';
+import { AppearanceOptions } from '@fluentui-react-native/theme-types';
+import { assertNever } from 'assert-never';
+
+export function getMacOSAliasTokens(mode: AppearanceOptions, isHighContrast: boolean) {
+  if (mode === 'light') {
+    if (isHighContrast) {
+      return macOSLightHCAliasTokens;
+    } else {
+      return macOSLightAliasTokens;
+    }
+  } else if (mode === 'dark') {
+    if (isHighContrast) {
+      return macOSDarkHCAliasTokens;
+    } else {
+      return macOSDarkAliasTokens;
+    }
+  } else if (mode === 'highContrast') {
+    throw new Error('highContrast is not a valid AppearanceOptions on macOS');
```

Comment:
nit: To clarify that we _do_ indeed support high contrast, maybe we can say something like "High Contrast must be enabled in system preferences to get High contrast styling"

Mostly so that the first time someone gets this error message, they don't report a bug "macOS doesn't support high contrast" incorrectly.